SkyWars/src/vixikhd/skywars/arena/ArenaScheduler.php

```diff
@@ -134,9 +132,9 @@ public function onRun(int $currentTick) {
     public function reloadSign() {
         if(!is_array($this->plugin->data["joinsign"]) || empty($this->plugin->data["joinsign"])) return;
 
-        $signPos = Position::fromObject(Vector3::fromString($this->plugin->data["joinsign"][0]), $this->plugin->plugin->getServer()->getLevelByName($this->plugin->data["joinsign"][1]));
+        $signPos = Position::fromObject(Vector3::fromString($this->plugin->data["joinsign"][0]), $this->plugin->plugin->getServer()->getWorldManager()->getWorldByName($this->plugin->data["joinsign"][1]));
 
-        if(!$signPos->getLevel() instanceof Level || is_null($this->plugin->level)) return;
+        if(!$signPos->getWorld() instanceof World || is_null($this->plugin->level)) return;
```

Comment:
Position->getWorld() no longer returns null and is always of World instance
```suggestion
        if(!$signPos->getWorld()->isLoaded() || is_null($this->plugin->level)) return;
```